test/tools/spec-runner/index.js

```diff
@@ -636,16 +649,22 @@ const kOperations = new Map([
     'createCollection',
     (operation, db, context /*, options */) => {
       const collectionName = operation.arguments.collection;
+      const encryptedFields = operation.arguments.encryptedFields;
       const session = maybeSession(operation, context);
-      return db.createCollection(collectionName, { session });
+      return db.createCollection(collectionName, { session, encryptedFields });
     }
   ],
   [
     'dropCollection',
     (operation, db, context /*, options */) => {
       const collectionName = operation.arguments.collection;
+      const encryptedFields = operation.arguments.encryptedFields;
       const session = maybeSession(operation, context);
-      return db.dropCollection(collectionName, { session });
+      return db.dropCollection(collectionName, { session, encryptedFields }).catch(err => {
+        if (err.code !== MONGODB_ERROR_CODES.NamespaceNotFound) {
+          throw err;
+        }
```

Comment:
The added spec tests are pretty clear about the fact that dropping a non-existent collection would not be expected to make the test fail. Let me know if this isn’t a good way to implement that behavior.

Reply:
I think checking the code is fine - it's been there before 3.6.